tests/integration/test_api.py

```diff
@@ -92,3 +93,31 @@ async def test_route():
     assert body["exception_id"] == exception_id
     assert body["description"] == description
     assert body["data"] == data
+
+
+async def test_request_duration_log(caplog):
+    """Check the middleware function that logs request duration in ms."""
+    caplog.set_level("INFO")
+
+    # small fastapi app with basic endpoint
+    test_app = FastAPI()
+    test_app.get("/")(lambda: 200)
+
+    config = ApiConfigBase()
+    configure_app(test_app, config)
+
+    client = AsyncTestClient(test_app)
+    await client.get("/")
+
+    # Get list of the log messages
+    for record in caplog.record_tuples:
+        if record[0] == "ghga_service_commons.api.api":
+            msg = record[2]
+            assert msg.endswith(" ms")
+            msg = msg.removesuffix(" ms")
+
+            duration = msg.rsplit(" ", 1)[1]
+            assert duration.isnumeric
```

Comment:
```suggestion
            assert re.match(r'GET http://.* "200 OK" - \d+ ms$', record[2])
```

Reply:
Thanks for the suggestion. See [e6c0a79](https://github.com/ghga-de/ghga-service-commons/pull/54/commits/e6c0a7964f63418cbe64ebace52dd67b0722a9ac)